### PR TITLE
prereqs and clearer installation steps

### PR DESCRIPTION
**Pull request by EdwardAngert**
Closes: DOC-266

## Summary

**[Drupal 9](https://pantheon.io/docs/drupal-9)** - Clarify installation steps and prerequisites to help less advanced users.

## Remaining Work
(Remove if not needed)
The following changes still need to be completed:

- [x] tech edit (could be combined with copy)?
- [x] copy edit

## Post Launch
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/drupal-9.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ Pantheon engineers are rolling out changes often.
 
 </Alert>
 
+## Before You Begin
+
+The steps in this doc require command line tools on your local machine. You'll also need to be logged in to your Pantheon account. If you don't have one, [create one first](https://pantheon.io/register?docs) and familiarize yourself with the [User Dashboard](/guides/quickstart/user-dashboard) before you create a new site.
```
Comment (alexfornuto) [inline source/content/drupal-9.md:24]: Is Terminus explicitly required for this?

Comment (EdwardAngert) [reply source/content/drupal-9.md:24]: good catch - I think of it as a "just do it while you're in the cli"
I'll remove that while I fix whatever the CI just caught

Comment (EdwardAngert) [reply source/content/drupal-9.md:24]: tagging @carolyn-shannon in on this - she picked up this into to make it a little less daunting

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | [Live](https://pantheon.io/docs) | [pr-6151](https://pr-6151--pantheon-docs.my.pantheonfrontend.website/docs) | Difference |
| --- | --- | --- | --- |
| Performance Score | `51` | `54` | `3` |
| Best Practice Score | `86` | `86` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `88` | `88` | `0` |

Lighthouse Audit Reports:

- [`pr-6151` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/25135/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/25135/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)


[Preview Environment](https://pr-6151--pantheon-docs.my.pantheonfrontend.website/docs)

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**